src/Autofac.Extras.Moq/MoqRegistrationHandler.cs

```diff
@@ -1,5 +1,5 @@
 // This software is part of the Autofac IoC container
-// Copyright (c) 2007 - 2008 Autofac Contributors
+// Copyright (c) 2020 Autofac Contributors
```

Comment:
Another that should probably be 2007 or 2007-2020. (Did we really have this in 2007?)

Reply:
Reverted. Also, I doubt it...it wouldn't have been much use without AutoMock, which is apparently 6 years newer?